RecoTracker/MkFit/data/mkfit-phase1-detachedQuadStep.json

```diff
@@ -0,0 +1,1321 @@
+{
+ "m_track_algorithm": 24,
+ "m_params": {
+  "nlayers_per_seed": 4,
+  "maxCandsPerSeed": 5,
+  "maxHolesPerCand": 4,
+  "maxConsecHoles": 1,
+  "chi2Cut": 30.0,
+  "chi2CutOverlap": 3.5,
+  "pTCutOverlap": 1.0,
+  "c_ptthr_hpt": 2.0,
+  "c_drmax_bh": 0.017999999225139618,
```

Comment:
could a comment be added in order to make these "magic" numbers a little bit more digestible ?

Reply:
in addition, do they really need 10^-17 precision !?!?

Reply:
Config parameter descriptions will be added to README.md

JSON is generated via nlohman::json functions that do not allow setting of precision and always prints floating point values as doubles (the parameters themselves are floats). We were considering whether to provide specialised JSON export functions or to post-process JSON to remove extra digits. But there is a benefit to having them in full-precision -- it allows us to detect if the value has changed from the default one and report number of changed parameters.